### PR TITLE
Migrate dnb03 mount point to dnb-ds03

### PR DESCRIPTION
**Pull request by domgz**
Remove dnb03 mount point.

Related: usegalaxy-eu/infrastructure-playbook#970.

### CODE REVIEW

**Comment by domgz** [comment]
See [this comment](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/970#issuecomment-1785159432).

**Comment by domgz** [comment]
To everybody, **remember to wait until Thursday to merge this**.

**Review by mira-miracoli — APPROVED** [review]
Looks good :)

**Pull request merged**
